build.sbt

```diff
@@ -36,7 +36,7 @@ addSbtPlugin(
 addSbtPlugin("com.github.jkinkead" % "sbt-scalariform" % "0.1.6")
 
 // Dependency graph visualiztion in SBT console
-addSbtPlugin("com.gilt" % "sbt-dependency-graph-sugar" % "0.7.4")
+addSbtPlugin("com.gilt" % "sbt-dependency-graph-sugar" % "0.7.5-1")
```

Comment:
If this isn't required by your plugin, you may want to remove (or put in a separate PR).